tic-tac-toe.js

```diff
@@ -1,7 +1,76 @@
 function TicTacToe() {
-
+  this.winConditions = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8],[0,4,8], [6,4,2]];
+  this.clickSpace();
 }
 
 TicTacToe.prototype = {
-  
-}
+  // check which player and return the appropriate mark
+  drawMark: function(player){
+    return player === 1 ? '🍍' : '🌺';
+  },
+
+  resetBoard: function(){
+    $(".square").each(function(){
+      this.innerText = "";
+    });
+  },
+
+  //check every square - if they are all filled, then do alert/game over TicTacToe
+  checkForTie: function() {
+   var self = this;
+   var filled = 0;
+   $(".square").each(function(){
+     if(this.innerText == ""){
+       filled++;
+     }
+   });
+   if (filled == 0){
+     swal({
+    title: "Game Over - It's a tie!",
+    text: "🍍🌺🍍🌺",
+    confirmButtonText: "x" });
+     self.resetBoard();
+   }
+  },
+
+  checkForWin: function(player){
+    var self = this;
+    var winConditions = this.winConditions;
+    var board = $(".square"); // == [r1c1,r1c2,r1c3,....]
+    for (var i = 0; i < winConditions.length; i++){
+      if((board[winConditions[i][0]].innerText == board[winConditions[i][1]].innerText)
```

Comment:
I think this is where an array that stored the board as you take turns would come in handy. I think you've successfully used just the UI/jQuery elements up until this point, but this conditional would be a lot more simple given an array which stores each players moves